src/modules/storage/LedgerStorage.ts

```diff
@@ -568,6 +582,139 @@ export class LedgerStorage extends Storages
             })();
         });
     }
+    /**
+     * Put BOA Account Stats.
+     * @param block: The instance of the `Block`
+     */
+    public putAccountStats (block: Block): Promise<void> {
+        function save_stats (storage: LedgerStorage, address: string, accountInfo: IAccountInformation, received_amount: JSBI, sent_amount: JSBI, height: Height): Promise<void> {
+            return new Promise<void>((resolve, reject) => {
+                storage.query(
+                    `INSERT INTO accounts
+                            (address, tx_count, total_received, total_sent, total_reward, total_frozen, total_spendable, total_balance, last_upated_at)
+                        VALUES
+                            (?, ?, ?, ?, ?, ?, ?, ?, ?)
+                        ON DUPLICATE KEY UPDATE 
+                                    address = VALUES(address), 
+                                    tx_count = IF(last_upated_at = VALUES(last_upated_at), tx_count, (tx_count + VALUES(tx_count))),
```

Comment:
Shouldn't `IF(last_upated_at = VALUES(last_upated_at),` be applied to the code in the 2 lines below?

Reply:
i didn't get your point please elaborate

Reply:
```suggestion
                                    tx_count = IF(last_updated_at = VALUES(last_updated_at), tx_count, (tx_count + VALUES(tx_count))),
```